File: ueueo-framework/ueueo-localization/src/main/java/Abp/Localization/LocalizationSettingHelper.cs
using JetBrains.Annotations;

namespace Volo.Abp.Localization;

public static class LocalizationSettingHelper
{
    /**
     * Gets a setting value like "en-US;en" and returns as splitted values like ("en-US", "en").
    *
     * <param name="settingValue"></param>
     * <returns></returns>
     */
    public static (String cultureName, String uiCultureName) ParseLanguageSetting(@Nonnull String settingValue)
    {
        Objects.requireNonNull(settingValue, nameof(settingValue));

        if (!settingValue.Contains(";"))
        {
            return (settingValue, settingValue);
        }

        var splitted = settingValue.Split(';');
        return (splitted[0], splitted[1]);
    }
}

File: abp-frameworks/ueueo-localization/src/main/java/com/ueueo/localization/LocalizationSettingHelper.java
package com.ueueo.localization;

import org.springframework.lang.NonNull;

import java.util.Objects;

public class LocalizationSettingHelper {

    private static final String LANGUAGE_SETTING_CULTURE_SPLIT_CHAR = ";";

    /**
     * Gets a setting value like "en-US;en" and returns as splitted values like ("en-US", "en").
     *
     * <param name="settingValue"></param>
     * <returns></returns>
     */
    public static String parseLanguageSettingCultureName(@NonNull String settingValue) {
        Objects.requireNonNull(settingValue);

        if (!settingValue.contains(LANGUAGE_SETTING_CULTURE_SPLIT_CHAR)) {
            return settingValue;
        }

        return settingValue.split(LANGUAGE_SETTING_CULTURE_SPLIT_CHAR)[0];
    }

    public static String parseLanguageSettingUiCultureName(@NonNull String settingValue) {
        Objects.requireNonNull(settingValue);

        if (!settingValue.contains(LANGUAGE_SETTING_CULTURE_SPLIT_CHAR)) {
            return settingValue;
        }

        return settingValue.split(LANGUAGE_SETTING_CULTURE_SPLIT_CHAR)[1];
    }
}
